### PR TITLE
Remove Console.WriteLine from Complex tests

### PR DESCRIPTION
**Pull request by hpshelton**
Remove Console.WriteLine from Complex tests.
More effectively use Asserts in Complex test code.

Progress against #915

### CODE REVIEW

**Comment by hpshelton** [comment]
Blocked on #1434

**Comment by ellismg** [comment]
@dotnet-bot test this please

**Comment by Maxwe11** [comment]
@dotnet-bot test this please

**Comment by hpshelton** [comment]
@ellismg - Per feedback, replaced CLR names with C# aliases, removed Support.RandomSampleCount, resolved spacing in the `ifdef` (it's definitely 8 or 12 spaces; VS appears to not autoformat blocks within non-applicable `ifdef`).

**Comment by mellinoe** [comment]
This looks good to me, but give me some time to look through it all. It's a bit hard to see the real code changes because of all the Double->double changes. It's probably too late to separate that into two commits, unfortunately :wink:

File: src/System.Runtime.Numerics/tests/Complex/ComplexTestSupport.cs [hunk 1]
```diff
@@ -11,14 +10,10 @@ namespace ComplexTestSupport
     public static class Support
     {
         private static Random s_random;
-        public static int randomSample;
```
Comment (mellinoe) [inline src/System.Runtime.Numerics/tests/Complex/ComplexTestSupport.cs]: Any particular reason you removed this? Seems that it was easier to tweak as a field

Comment (hpshelton) [inline src/System.Runtime.Numerics/tests/Complex/ComplexTestSupport.cs]: @ellismg made the opposite feedback earlier and I agree with him - Support.RandomSampleCount doesn't mean '3' to me, and it was only used in a half-dozen places, so better to just inline it where it's used.

**Comment by hpshelton** [comment]
Any further comments here, folks?

**Comment by mellinoe** [comment]
Looks good, thanks again.

**Pull request merged**